src/drivers/driver_commands.cpp

```diff
@@ -30,6 +32,15 @@ etl::optional<int> handle_command(DriverCommand type, int arg1, int arg2, int ar
                 static_cast<uint32_t>(arg1)
             );
             break;
+        case DriverCommand::SET_PIN:
+            edge::aidan::set_gpio_pin(static_cast<uint32_t>(arg1));
+            break;
+        case DriverCommand::CLEAR_PIN:
+            edge::aidan::clear_gpio_pin(static_cast<uint32_t>(arg1));
+            break;
+        case DriverCommand::SET_PIN_OUTPUT:
+            GPIOPin(static_cast<uint32_t>(arg1), GPIOConfiguration::OUT);
```

Comment:
This feels like a really weird way to do this...

Side note (I'm willing to do this), it could be cool to have `GPIOPin` be a userspace object that is returned by a syscall

Reply:
we could but doesnt seem needed and i dont love expoisng gpio to userspace if we want to build abstraction, cc these syscalls really just being here for illustration